tools/cockpit.spec

```diff
@@ -143,6 +143,7 @@ Requires: cockpit-system
 # Optional components
 Recommends: (cockpit-storaged if udisks2)
 Recommends: (cockpit-packagekit if dnf)
+Recommends: (cockpit-ostree if rpm-ostree)
```

Comment:
Hmm, wait -- as long as the package is not in RHEL, I think we should move this to the section below. We can move it back if/when it gets into RHEL.

Reply:
right, but should not hurt, right? Surely dnf can deal with recommends that do not exists in any repo.

Reply:
well, https://bugzilla.redhat.com/show_bug.cgi?id=2093231 …